mmv1/products/storage/ManagedFolder.yaml

```
@@ -0,0 +1,59 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'ManagedFolder'
+kind: 'storage#ManagedFolder'
+base_url: b/{{bucket}}/managedFolders
+self_link: b/{{bucket}}/managedFolders/{{managedFolder}}
+references: !ruby/object:Api::Resource::ReferenceLinks
+  guides:
+    'Official Documentation': 'https://cloud.google.com/storage/docs/json_api/v1/managedFolder/insert'
+  api: 'https://cloud.google.com/storage/docs/json_api/v1/managedFolder'
+description: |
+  The ManagedFolder resource represents a managed folder in Cloud Storage.
+  You can apply IAM policies on a managed folder to control access to the resources within the managed folder.
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'google_storage_managed_folder'
+    primary_resource_id: 'managed_folder'
+    vars:
+      bucket_name: 'static-content-bucket-03'
+      name: "managed-folder-name"
+id_format: '{{bucket}}/{{managedFolder}}'
+import_format: ['{{bucket}}/{{managedFolder}}']
+mutex: 'storage/buckets/{{bucket}}'
+immutable: true
+skip_sweeper: true
+iam_policy: !ruby/object:Api::Resource::IamPolicy
+  parent_resource_attribute: 'managedFolders'
+  method_name_separator: ':'
```

It looks like this API uses `/` in IAM URLs - that is, the URLs are like:

```
https://storage.googleapis.com/storage/v1/b/bucket/managedFolders/{{managedFolder}}/iam
```

Instead of like:

```
https://apigee.googleapis.com/v1/{resource=organizations/*/environments/*}:getIamPolicy
```


```suggestion
  method_name_separator: '/'
```

---

fix in last commit